### PR TITLE
Update node checks

### PR DESCRIPTION
**Pull request by amanda11**
Update to check against node  > 12

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -19,4 +21,6 @@ matrix:
     - node_js: "7"
     - node_js: "11"
     - node_js: "12"
+    - node_js: "13"
+    - node_js: "14"
```
Comment (blag) [inline .travis.yml:25]:
```suggestion
```

We should force these to pass from now on.

**Review by blag — CHANGES_REQUESTED** [review]
Looks good, although I'd like to minimize our test matrix as well (eg: remove tests for Node < 10) and force us to pass on more up-to-date pinned Node versions.

**Comment by amanda11** [comment]
@blag - thanks for review. Shorted test matrix, am I ok to leave hubot-stackstorm version as before as not changed anything in the actual package?

**Comment by blag** [comment]
For this change, that's correct - we do not need to bump and push a new version to NPM, as this is just testing changes.

If we update dependencies, like all of @dependabot's PRs, we should bump the version. But we can "batch" those together with respect to versioning - in other words, we can merge in multiple dependency updates before we bump the version and push it to NPM.

**Review by blag — APPROVED** [review]

**Pull request merged**